### PR TITLE
TICK Monitoring Stack

### PR DESCRIPTION
**Pull request by kulinacs**

### CODE REVIEW

**Comment by the-maldridge** [comment]
@kulinacs please contact me OOB to discuss this PR.

File: srcpkgs/influxdb/template [hunk 1]
```diff
@@ -1,46 +1,41 @@
...
-build_style=go
-hostmakedepends="git-perl mercurial"
-go_import_path="github.com/influxdata/influxdb"
-go_package="${go_import_path}/cmd/influx
- ${go_import_path}/cmd/influxd
- ${go_import_path}/cmd/influx_inspect
- ${go_import_path}/cmd/influx_stress
- ${go_import_path}/cmd/influx_tsm"
+version=1.3.1
+revision=6
```
Comment (Vaelatern) [inline srcpkgs/influxdb/template]: no

File: srcpkgs/influxdb/template [hunk 2]
```diff
@@ -1,46 +1,41 @@
...
+_branch=1.3
+_commit=5887e92
+build_style="go"
```
Comment (Vaelatern) [inline srcpkgs/influxdb/template]: You added quotes where not needed.

File: srcpkgs/influxdb/template [hunk 3]
```diff
@@ -1,46 +1,41 @@
...
+nocross=1
```
Comment (Vaelatern) [inline srcpkgs/influxdb/template]: Reason?

File: srcpkgs/telegraf/files/telegraf.conf [hunk 4]
```diff
@@ -0,0 +1,2477 @@
+# Telegraf Configuration
```
Comment (Vaelatern) [inline srcpkgs/telegraf/files/telegraf.conf]: Please lose this file. Perhaps there can be a good configuration file found elsewhere, but I don't want this file in our repo.

File: srcpkgs/influxdb/template [hunk 5]
```diff
@@ -1,46 +1,41 @@
...
+go_import_path="github.com/influxdata/${pkgname}"
```
Comment (Vaelatern) [inline srcpkgs/influxdb/template]: no need to add a variable here, especially when the variable is longer than the string it replaces.

File: srcpkgs/influxdb/template (hunk 3 above)
Comment (kulinacs) [inline srcpkgs/influxdb/template]: https://travis-ci.org/kulinacs/void-packages/jobs/258683433

Comment (the-maldridge) [inline srcpkgs/influxdb/template]: 1) That doesn't look like the cross build is actually failing.
2) If the cross build is actually failing, set nocross to be the link of the failing build log (ideally plaintext) as xbps-src will dump the contents of that variable into the logs for why this package wasn't cross built.

**Comment by kulinacs** [comment]
Requested changes have been made.

**Pull request closed without merging**